app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+    
+  def create
+    job = Job.find(params[:job][:id])
+    company = job.company
+    @comment = Comment.new(comment_params)
+    @comment.job_id = job.id
```

Comment:
we talked about this in class - how can we leverage our associations to avoid having to manually set this foreign key?